bitnami/kubeapps/README.md

```diff
@@ -95,70 +95,70 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
-| `frontend.extraVolumes`                          | Optionally specify extra list of additional volumes for frontend pods                     | `[]`                    |
-| `frontend.extraVolumeMounts`                     | Optionally specify extra list of additional volumeMounts for frontend container(s)        | `[]`                    |
-| `frontend.sidecars`                              | Add additional sidecar containers to the frontend pod                                     | `{}`                    |
-| `frontend.initContainers`                        | Add additional init containers to the frontend pods                                       | `{}`                    |
-| `frontend.service.type`                          | Frontend service type                                                                     | `ClusterIP`             |
-| `frontend.service.port`                          | Frontend service HTTP port                                                                | `80`                    |
-| `frontend.service.nodePort`                      | Node port for HTTP                                                                        | `nil`                   |
-| `frontend.service.clusterIP`                     | Frontend service Cluster IP                                                               | `nil`                   |
-| `frontend.service.loadBalancerIP`                | Frontend service Load Balancer IP                                                         | `nil`                   |
-| `frontend.service.loadBalancerSourceRanges`      | Frontend service Load Balancer sources                                                    | `[]`                    |
-| `frontend.service.externalTrafficPolicy`         | Frontend service external traffic policy                                                  | `Cluster`               |
-| `frontend.service.annotations`                   | Additional custom annotations for frontend service                                        | `{}`                    |
+| Name                                             | Description                                                                               | Value                  |
+| ------------------------------------------------ | ----------------------------------------------------------------------------------------- | ---------------------- |
+| `frontend.image.registry`                        | NGINX image registry                                                                      | `docker.io`            |
+| `frontend.image.repository`                      | NGINX image repository                                                                    | `bitnami/nginx`        |
+| `frontend.image.tag`                             | NGINX image tag (immutable tags are recommended)                                          | `1.20.1-debian-10-r32` |
```

Comment:
So it looks like the only meaningful differences in this file is the updating of the documented default values for image tags, and some of them are incorrect for reasons I'll explain in the later. So we'll re-generate this after updating correcting the change in the values.yaml. (eg. the nginx image tag should be 1.21)